extension/src/content.ts

```diff
@@ -1,3 +1,24 @@
+{
+}
+
```

Comment:
This is odd, I think it can be removed.

Reply:
You can't remove it. If you do weird stuff will start to happen (Plasmo Docs etc).

Reply:
LOL